apps/community/src/app/member/professor/page.tsx

```diff
@@ -0,0 +1,26 @@
+import { PageHeader } from '~/components/page-header';
+import { ProfessorCard } from '~/components/professor-card';
+
+import * as styles from '~/app/member/professor/page.css';
+
+import { PROFESSORS } from './mocks/professor';
+
+export default function Page(): JSX.Element {
```

Comment:
```suggestion
export default function Professor() {
```
or `ProfessorPage`

현재 root page가 `Home`으로 되어 있어, `Professor`로 지정하는게 현 상황에는 일관성 있을거 같어요

Reply:
page는 일관성을 위해 prefix + page -> `ProfessorPage`로 통일하는게 어떨까요 ?

Reply:
좋아요! 페이지임을 명시하면서 어떤 페이지인지 의미가 부여되면 보기 좋을 것 같아요.